File: patch_ventures.py
import sys

with open("src/components/home/Ventures.jsx", "r") as f:
    content = f.read()

# Remove the Strategic Assets block
# The block starts with `<div className="mt-32 mb-16 text-center">`
# and ends right before `</motion.div>\n    </section>`

start_index = content.find('<div className="mt-32 mb-16 text-center">')
end_index = content.rfind('</motion.div>\n    </section>')
# Since we have two `</motion.div>` there, the first rfind will find the very last `</motion.div>` which belongs to the parent.
# Wait, let's carefully replace using regex or string replace.

import re

# Remove handleAssetDownload
content = re.sub(r'\s*const handleAssetDownload = \(assetId\) => \{\s*console\.log\(`Prepared to download asset: \$\{assetId\}`\);\s*\};\s*', '\n\n', content)

# Remove the strategic assets section
content = re.sub(r'<div className="mt-32 mb-16 text-center">.*?STRATEGIC.*?<span className="text-electric-gold">ASSETS</span></h2>\s*</div>\s*<motion\.div variants=\{containerVariant\}.*?className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-8">.*?\]\.map\(\(asset, idx\) => \(.*?</motion\.div>\s*\)\)\}\s*</motion\.div>', '', content, flags=re.DOTALL)

with open("src/components/home/Ventures.jsx", "w") as f:
    f.write(content)

print("Patched Ventures.jsx")

File: patch_ventures.js
const fs = require('fs');

const path = 'src/components/home/Ventures.jsx';
let content = fs.readFileSync(path, 'utf8');

const digitalAssetsSection = `
        <div className="mt-32 mb-16 text-center">
          <span className="font-editorial text-[10px] text-gray-500 uppercase tracking-widest font-bold block mb-4">Digital Products</span>
          <h2 className="font-editorial font-black text-4xl text-white">STRATEGIC <span className="text-electric-gold">ASSETS</span></h2>
        </div>

        <motion.div variants={containerVariant} initial="hidden" whileInView="show" viewport={{ once: true, margin: "-100px" }} className="grid lg:grid-cols-2 gap-8">
          {[
            {
              title: "The Blue-Collar Automation Dividend Manual",
              description: "A comprehensive guide to algorithmic economic equity and implementing the Automation Dividend in local economies.",
              tag: "E-Book / Manual"
            },
            {
              title: "Sovereign Supply Chains Compendium",
              description: "Strategic frameworks for decentralized logistics, vital system protection, and building localized supply networks.",
              tag: "Strategic Framework"
            }
          ].map((asset, idx) => (
            <motion.div variants={itemVariant} key={idx} className="will-change-transform">
              <div className="bg-void border border-white/10 rounded-sm p-10 flex flex-col h-full group hover:border-white/20 transition-colors deco-brackets relative">
                <span className="font-editorial text-[10px] text-yellow-electric uppercase tracking-widest font-bold block mb-4">{asset.tag}</span>
                <h3 className="font-editorial font-black text-2xl text-white mb-4 uppercase tracking-tight">{asset.title}</h3>
                <p className="text-text-muted leading-relaxed font-light flex-grow mb-8">{asset.description}</p>
                <div className="mt-auto">
                  <button className="border border-yellow-electric/30 text-yellow-electric text-xs tracking-widest uppercase hover:bg-yellow-electric/10 transition-colors px-6 py-3 w-full sm:w-auto">
                    Acquire Asset
                  </button>
                </div>
              </div>
            </motion.div>
          ))}
        </motion.div>
`;

content = content.replace('</motion.div>\n      </motion.div>\n    </section>', '</motion.div>\n' + digitalAssetsSection + '      </motion.div>\n    </section>');
fs.writeFileSync(path, content);
